templates/rails/app/views/layouts/application.html.erb.tt

```diff
@@ -47,11 +47,5 @@
   <%%= yield %>
 
   <%%= javascript_include_tag 'application' %>
-  <%%= javascript_tag do %>
-    $(function(){
-      $(document).foundation();
-    });
-  <%% end %>
-  <%%= yield :extra_footer %>
```

Comment:
I think we should still include the `yield :extra_footer` as it allows users to inject javascript at the bottom of the page